src/WebUI/Resources.js

```diff
@@ -3,7 +3,7 @@
 define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], function (require, exports) {

```

Comment:
in this file, please take care of

"Associated Pods" also as "Associated pods"

Reply:
Done